.semaphore/semaphore.yml

```diff
@@ -0,0 +1,37 @@
+---
+version: "v1.0"
+name: "Vaultenv CI Pipeline"
+
+agent:
+  machine:
+    type: "e1-standard-2"
+    os_image: "ubuntu1804"
```

Comment:
We could try using Semaphore's more recent `ubuntu2004` image, since the core of the pipeline is run using Nix anyway.

Reply:
:+1: